src/Apps/__stories__/OrderApp.story.tsx

```diff
@@ -33,16 +42,27 @@ storiesOf("Apps/Order Page", module)
   ))
   .add("Shipping - Untouched Order", () => (
     <Router
-      // The UntouchedOrder has a specified requestedFulfillment, but it should be null.
-      // Unfortunately, enough of our tests use UntouchedOrder to change it, so we'll specify it here to avoid breaking our story.
+      // The UntouchedBuyOrder has a specified requestedFulfillment, but it should be null.
+      // Unfortunately, enough of our tests use UntouchedBuyOrder to change it, so we'll specify it here to avoid breaking our story.
       mockResolvers={mockResolver({
-        ...UntouchedOrder,
+        ...UntouchedBuyOrder,
         requestedFulfillment: null,
       })}
       initialRoute="/orders/123/shipping"
     />
   ))
   .add("Review", () => <Router initialRoute="/orders/123/review" />)
+  .add("Empty", () => (
```

Comment:
This seems to be a duplicate of the story above on line 27: "Apps/Order Page/Offer"?